docs/config.md

```diff
@@ -221,5 +221,12 @@ improvement, as well as a reduction in memory usage.
 - `DisableNatPortMap`
 Disable NAT discovery.
 
+- `DisableRelay`
+Disables the p2p-circuit relay transport.
```

Comment:
As discussed on IRC, I misunderstood what these toggles did. DisableRelay disables relaying entirely and we don't yet support advertising relayed addresses; when implemented, that will probably have an additional toggle.